services/web/server/src/simcore_service_webserver/meta.py

```diff
@@ -0,0 +1,33 @@
+""" Meta-modeling app module
+
+    Manages version control of studies, both the project document and the associated data
```

Comment:
is there not already a _meta.py module? just wondering if it would not gain a bit of clarity if it is called meta-something.py instead...

Reply:
The ``setup`` module is by convention  used as prefix of all the plugin's modules. Do you have any suggestions? ``meta_modeling_*``, ``meta_md_*``?

An alternative is changing ``_meta.py`` by something like ``_info.py`` which personally I prefer. :-)

Any preference?

Reply:
went for ``metaml_*.py``